.github/workflows/releaser.yml

```diff
@@ -160,6 +160,13 @@ jobs:
           GH_ARGS: --repo=${{ github.server_url}}/${{ github.repository }}
           GH_TOKEN: ${{ github.token }}
 
+      - name: Checkout the repository to update the nightly tag
```

Comment:
I'd rather merge this with the next item given they work closely together (and they are both behind the same `if env.NIGHTLY_RELEASE`)

Reply:
The action checkout also manage the authentication to the github repo using the provided github token.
Using the action is trivial, so I will keep it that way

Reply:
Yeah that makes sense :+1: